x-pack/plugins/actions/server/lib/task_runner_factory.ts

```diff
@@ -187,9 +188,9 @@ export class TaskRunnerFactory {
           logger.error(`Action '${actionId}' failed: ${e.message}`);
           if (e instanceof ActionTypeDisabledError) {
             // We'll stop re-trying due to action being forbidden
-            throwUnrecoverableError(e);
+            throwUnrecoverableError(e, TaskErrorSource.USER);
```

Comment:
Hmmm ... this actually isn't an error we even want to track as user or framework - from the names, it looks like this would get thrown when the connector type has been disabled - either via config, or perhaps some other way (license?).  Why we don't figure this out BEFORE we run, ??? :-).  

It's not really a user-configuration error, or a framework error.  It's kind of expected, if you have specifically disabled the connector type - not sure about the license aspect though, if that's involved.  I wonder if this is a call for another "class" of error alongside user/framework - "expected"?

Reply:
Yes it's both disabled action type and expired, invalid licence...
I think `TaskErrorSource.CONNECTOR_TYPE` would be better in our case